ow_lander/README.md

```diff
@@ -1,84 +1,149 @@
 
-### Trajectory feeding
-Call the PublishTrajectory service.
+* Stow
+```bash
+rosservice call /arm/stow "{}"
+```
+* Unstow
+```bash
+rosservice call /arm/unstow "{}"
+```
+* Deliver Sample
+```bash
+rosservice call /arm/deliver_sample \
+  "{use_defaults: true,
+    x: 0.0, y: 0.0, z: 0.0}"
+```
+* Dig Linear
+```bash
+rosservice call /arm/dig_linear \
+  "{use_defaults: true,
+    x: 0.0, y: 0.0, depth: 0.0,
+    length: 0.0, ground_position: 0.0}" 
+```
+* Dig Circular
+```bash
+rosservice call /arm/dig_circular \
+  "{use_defaults: true,
+    x: 0.0, y: 0.0, depth: 0.0,
+    parallel: false, ground_position: 0.0}"
+```
+* Grind
+```bash
+rosservice call /arm/grind \
+  "{use_defaults: true,
+    x: 0.0, y: 0.0, depth: 0.0,
+    length: 0.0,
+    parallel: false, ground_position: 0.0}" 
+```
+* Guarded Move
+```bash
+rosservice call /arm/guarded_move \
+  "{use_defaults: true,
+    x: 0.0, y: 0.0, z: 0.0,
+    direction_x: 0.0, direction_y: 0.0, direction_z: 0.0,
+    search_distance: 0.0}"
+```
 
 For more info on the message type of each of the services above, please refer to
 the message declaration in the .srv files. Alternatively, info can be displayed
 directly from terminal typing the following command:
-```
-$ rossrv show -r foo
+```bash
+rossrv show -r foo
 ```
 replacing "foo" with the desired service name (for instance, foo=DigCircular).
 
-### Manual Operations
-To run these services manually, find the 'rqt' window and select the Service
-Caller tab. Then, from the drop down menu, select the service you want
-(`/planning/arm/dig_circular` or `/planning/arm/publish_trajectory`), set the
-arguments (Careful, the bools have a capital first letter: `True`, `False`),
-then call the service.
-
-### MoveIt Setup Assistant
+## MoveIt Setup Assistant
 The moveit setup assistant can now be easily launched using the command:
 ```bash
 roslaunch ow_lander setup_assistant.launch
-``` 
+```
+
+Once the assistant is launched call the **Load Files** to load the current configuration.
+The following image shows a screenshot of a successfully loaded lander configuration:
+<p align="center"><img src="./misc/setup_assistant.png" width="40%" ></p>
 
-There are different reasons why you may need to re-run the setup assistant:
+There are various reasons why you may need to re-run the setup assistant:
 * Update the links collision matrix: this should be reconfigured again whenever
  links count, their shape or size changes or there was a change in the number of
  joints or their types.
 * Define new arm/antenna poses or revise existing ones.
 
 
+
+## Misc Information
+
+### Common Code
+`include/ow_lander/lander_joints.h` contains joint names as they appear in the
+URDF and related code. Please prefer this header over writing similar code
+yourself to keep all our code consistent. If the lander is modified to change
+the number of joints or any joint names, this header must be updated as well.
+
+### Package History
```

Comment:
Is the Package History section really helpful, especially in the public code?  Are any users likely to care or be impacted by the changes?

Reply:
I am with you one on this. I wasn't sure if other think it is helpful or not, personally I didn't read much through it, it seemed like our meeting notes. I am in favor of removing it.

Reply:
It's a good practice for automatically generated files to begin with a comment saying they are automatically generated. MoveIt Setup Assistant generates files that do not contain such a comment. This part of the README contains information about which files were generated and which of those have been modified by hand, which could be useful information if we need to re-run Setup Assistant. It should be revised due to all the changes in ow_lander, but probably not deleted.

Reply:
Terry, that's not the case here. This is a wizard generated files and they serve as a template but we are free to modify. What you are referring to are auto-generated that we shouldn't modify.

Reply:
In any case, I left a link to this OCEANWATER-518 PR that we can refer to later in case we needed to.

Reply:
If we re-generate files, any mods we made would be stomped. It's good to have an understanding of what mods we made so they can be added back in. Of course, it's easiest if we can avoid modifying generated files.

Reply:
Setup assistant does that for you. I mean it shows you which files are new and which files are going to be modified before you save the new config, plus all of our files are version tracked using git.